src/test/components/__snapshots__/TooltipMarker.test.js.snap

```diff
@@ -2057,38 +2057,38 @@ exports[`TooltipMarker renders tooltips for various markers: GCMajor-16.5 1`] =
     <div
       class="tooltipLabel"
     >
-      Phase minor_gc.mark_roots.mark_stack
```

Comment:
Yes, it looks right. I will note that it's slightly confusing (to me) because it preserves the execution order of phases, but the test data does *not* use the execution order. It uses alphabetical order instead. Which takes it a little bit away from the actual data it will be seeing, but it makes it much easier to verify that the snapshot is correct without knowing the inner details of the engine. I'm inclined to leave it alphabetical.